Dockerfile

```diff
@@ -1,10 +1,7 @@
 FROM debian:jessie
 
-# Default configuration
-COPY mopidy.conf /var/lib/mopidy/.config/mopidy/mopidy.conf
-
-# Start helper script
-COPY entrypoint.sh /entrypoint.sh
+# Set the username
+ENV UNAME user
```

Comment:
Why not have it within the RUN below? Or even just inline it and keep `mopidy` as it's clearer that user `user` which feels weirdly redundant.